### PR TITLE
Add MrSlimCoder

### LINKED ISSUE #607: MrSlimCoder
image

### PR DESCRIPTION
**Pull request by MRslimcoder**
Resolve #607

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
LGTM. Thank you for your contribution! 😄:+1:
I would only suggest adding `Resolve` in your comment before the issue number.

File: data.json [hunk 1]
```diff
@@ -1162,5 +1162,11 @@
...
     "imageUrl": "https://user-images.githubusercontent.com/70807684/126078076-3df641ea-3c0f-483f-952c-cbabbb5758c1.png",
     "issueNumber": 594
-  }
+  },
+  {
+    "name": "Slim Coder",
+    "githubUsername": "MrSlimCoder",
+    "imageUrl": "https://user-images.githubusercontent.com/28563357/126220805-82470ad3-f38a-40b4-a277-8c40704fdb5b.png",
+    "issueNumber": 607
+}
```
Comment (schmelto) [inline data.json:1171]:
```suggestion
  }
```

**Review by schmelto — CHANGES_REQUESTED** [review]

**Comment by MRslimcoder** [comment]
Team, test is still failed can you please verify it' related to linting issue @schmelto @Panquesito7

**Review by schmelto — APPROVED** [review]
lgtm

**Comment by schmelto** [comment]
> Team, test is still failed can you please verify it' related to linting issue @schmelto @Panquesito7 

I check the action issue later

**Review by Panquesito7 — APPROVED** [review]
LGTM. Thank you! 😄👍

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :rocket: Thanks for collaborating and contributing :)

**Pull request merged**